src/main/java/seedu/modtracker/ModuleList.java

```diff
@@ -130,6 +130,33 @@ public boolean checkIfExpTimeValid(double hours, boolean toPrint) {
         }
     }
 
+    /**
+     * Checks if the week is valid.
+     *
+     * @param week    number of weeks typed in by user.
+     * @param toPrint whether the UI should print the output.
+     * @return true if week number is valid, false otherwise.
+     */
+    public boolean checkIfWeekValid(String week, boolean toPrint) {
+        char c = week.charAt(0);
```

Comment:
should this be changed to checking both characters as week number can be 2 digits?